build_tools/test_frontend.sh

```diff
@@ -17,10 +17,16 @@ everware-server ${OPTS} > $LOG 2>&1 &
 HUB_PID=$!
 sleep 3
 
+if [ `pgrep -f jupyterhub` == "" ] ; then echo "Error starting" && exit 1 ; fi
```

Comment:
Do you know why this still works? Since we have `everware-server` now I would not expect this to find anything useful